src/citrine/informatics/design_spaces/design_space.py

```diff
@@ -35,3 +37,10 @@ def get_type(cls, data) -> Type[Serializable]:
             'FormulationDesignSpace': FormulationDesignSpace,
             'DataSourceDesignSpace': DataSourceDesignSpace
         }[data['config']['type']]
+
+    @property
+    def sample_design_space_executions(self):
```

Comment:
I wasn't sure the pattern to use here. On one hand, just calling sample would be awesome. But that kind of hides what is going on. So I named it this way.

Reply:
We do both in different classes, so it's your call.